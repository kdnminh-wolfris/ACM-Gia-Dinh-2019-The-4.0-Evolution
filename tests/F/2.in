8197 2426 4391 2645