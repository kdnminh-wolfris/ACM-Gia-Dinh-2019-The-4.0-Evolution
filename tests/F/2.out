5771 8197